src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Collections.Generic;
+
+namespace Stryker.Core.Mutants
+{
+    public static class MutantPlacer
+    {
+        public static IfStatementSyntax PlaceWithIfStatement(StatementSyntax original, StatementSyntax mutated, int mutantId)
+        {
+            // place the mutated statement inside the if statement
+            return SyntaxFactory.IfStatement(
+                condition: GetBinaryExpression(mutantId),
+                statement: SyntaxFactory.Block(mutated),
+                @else: SyntaxFactory.ElseClause(SyntaxFactory.Block(original)))
+                // Mark this node as a MutationIf node. Store the MutantId in the annotation to retrace the mutant later
+                .WithAdditionalAnnotations(new SyntaxAnnotation("MutationIf", mutantId.ToString()));
+        }
+
+        public static SyntaxNode RemoveByIfStatement(SyntaxNode node)
+        {
+            if (node is IfStatementSyntax ifStatement)
+                return ifStatement.Else.Statement;
+            else return null;
+        }
+
+        public static ConditionalExpressionSyntax PlaceWithConditionalExpression(ExpressionSyntax original, ExpressionSyntax mutated, int mutantId)
+        {
+            // place the mutated statement inside the if statement
+            return SyntaxFactory.ConditionalExpression(
+                condition: GetBinaryExpression(mutantId),
+                whenTrue: mutated,
```

Comment:
This and/or the conditional expression as a whole will run into [operator precedence](https://docs.microsoft.com/en-us/cpp/c-language/precedence-and-order-of-evaluation?view=vs-2017#precedence-and-associativity-of-c-operators) issues.  We should add parentheses to avoid this.

Reply:
What do you mean by that? For me no issues occured without the parentheses. In what case would this be better?

Reply:
After looking into this further, I'm not sure I can actually come up with an example that breaks.  I think I had the table reversed in my head.

I think that, because we're manipulating the syntax tree, we get some protection from this for free.